node_filesystem/templates/README.md

```diff
@@ -0,0 +1,46 @@
+The following is a summary of the templates defined in this directory:
+
+### bird.toml.template / bird6.toml.tempate
+
+Referenced by bird.toml.toml and bird6.toml.toml.
+
+These templates write out a TOML file (bird.toml and bird6.toml) that is used 
+to tell confd which set of the main BIRD templates to use.
+
+Based off the node_mesh parameter, the TOML file generated either points to the
+full-mesh config files, or the no-mesh config files.
+
+Once confd writes out the appropriate TOML file, confd is restarted to pick up
+the change to the bird.toml and bird6.toml files.  Since there are two sets
+of changes, a change to the node_mesh parameter will result in a double
```

Comment:
I don't understand this double restart of confd thing? Won't it restart confd then confd will restart bird?

Ah! Or do you mean one restart for ipv4 and one for ipv6?